component/file_cache/file_cache_test.go

```diff
@@ -689,7 +689,7 @@ func (suite *fileCacheTestSuite) TestCloseFileTimeout() {
 	defer suite.cleanupTest()
 	suite.cleanupTest() // teardown the default file cache generated
 	cacheTimeout := 5
-	config := fmt.Sprintf("file_cache:\n  path: %s\n  offload-io: true\n  timeout: %d\n\nloopbackfs:\n  path: %s",
+	config := fmt.Sprintf("file_cache:\n  path: %s\n  offload-io: true\n  timeout-sec: %d\n\nloopbackfs:\n  path: %s",
```

Comment:
there are couple of more occurrences which needs fix in this.

Reply:
Is this thing solving file-cache flaky UT failures?

Reply:
I ran the pipeline a couple times and didnt see it repro but before i saw it at least in one distro each run

Reply:
Ok I fixed it in the other places